server/routes/api/trips/tripRoutes.js

```diff
@@ -24,20 +26,59 @@ router.post(
 
 router.get(
   '/cities',
+  authorizationCheck,
   allow('Requester', 'Manager'),
   tripController.getAllCities,
 );
 
 router.patch(
   '/:trip_id',
   [
-    allow('Requester', 'Manager'),
     authorizationCheck,
+    allow('Requester', 'Manager'),
     tripValidations.updateTripJoiValidation,
     tripVerification,
     tripValidations.checkOpenTripCityAndDATE,
   ],
   tripController.updateOpenTripDetails,
 );
 
+router.get('/', 
+    [
+      authorizationCheck,
+      allow('Requester', 'Manager'),
+      tripValidations.allTripsJOIValidation,
+      tripValidations.paginationByUserId,
+    ],
+    tripController.viewAllTrips,
+    );
+
+router.get('/assigned', 
+  [
+    authorizationCheck,allow('Manager'),
+    tripValidations.allTripsJOIValidation,
+    tripValidations.paginationByLineManagerId,
+  ],
+    tripController.viewAllTrips,
+    );
+
+router.get('/assigned/:trip_id', 
```

Comment:
What is the difference between trips/assigned/:trip_id and trips/:trip_id?

Reply:
`/:trip_id`: is used when the user/manager wants to view all the trip requests he/she made,
but 
`/assigned/:trip_id`: is used when the manager wants to view all the trip requests assigned to him or her.